o4/o4_prune.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+from time import time
+
+from o4_fstat import prune_fstat_cache
+
+
+def prune_data(o4_dir):
+    prune_fstat_cache(o4_dir)
+    prune_cleaned(o4_dir)
+
+
+def prune_cleaned(o4_dir):
+    '''
+    Remove all read-only files from an abandoned cleaned directory
+    (unless it's too recent).
+    '''
+    cleaned = Path(o4_dir) / 'cleaned'
+    if not cleaned.exists():
+        return
+    if not cleaned.is_dir():
+        print('*** WARNING: "cleaned" is not a directory! Not trying to prune it.')
+        return
+    age = time() - cleaned.stat().st_mtime
+    if age < 86400 * 2:
+        print('*** INFO: You have files left behind by a recent "clean" operation.')
+        print('          It will be pruned soon.')
+        return
+    print('*** INFO: Pruning abandoned "cleaned" directory')
+    prune_item(cleaned)
+    if not cleaned.exists():
+        return
+    n_left = sum(1 for f in cleaned.glob('**/*') if not f.is_dir())
```

Comment:
I think you need `, recursive=True` for `**` to work

Reply:
oh Path.glob, not glob.glob, not sure.

Reply:
The ** makes it recursive.